### ISSUE TITLE
Bad performance on validation - new datacube version

### ISSUE THREAD

**Issue by initze** [initze/thaw-slump-segmentation#70]
The new datacube version training has bad performance

Training  looks alright, though not great
IoU and F1 = 0

![grafik](https://github.com/initze/thaw-slump-segmentation/assets/4864803/6361d228-5211-4bb3-96fa-c63f9671ee5e)
![grafik](https://github.com/initze/thaw-slump-segmentation/assets/4864803/d07d5581-ab6b-4174-a606-207cb63608d0)

### Config
```
# Model Specification
model:
  # Model Architecture. Available:
  # Unet, UnetPlusPlus, Unet3Plus, MAnet, Linknet, FPN, PSPNet, DeepLabV3, DeepLabV3Plus, PAN]
  architecture: UnetPlusPlus
  # Model Encoder. Examples:
  # resnet18, resnet34, resnet50, resnet101, resnet152
  # Check https://github.com/qubvel/segmentation_models.pytorch#encoders for the full list of available encoders
  encoder: resnet34
  # Encoder weights to use (if transfer learning is desired)
  # `imagenet` is available for all encoders, some of them have more options available
  # `random` initializes the weights randomly
  # Check https://github.com/qubvel/segmentation_models.pytorch#encoders for the
  # full list of weights available for each encoder
  encoder_weights: random
# Loss Function to use. Available:
# JaccardLoss, DiceLoss, FocalLoss, LovaszLoss, SoftBCEWithLogitsLoss
loss_function: SoftBCEWithLogitsLoss
# Data Configuration
data_threads: 8  # Number of threads for data loading, must be 0 on Windows
tile_size: 1024 # tile size in pixels
# Sampling mode to Use. Available:
# deterministic, random, targets_only
sampling_mode: deterministic
data_sources:  # Enabled input features
  - PlanetScope
  - TCVIS
  #- RelativeElevation
  - AbsoluteElevation
  - Slope
datasets:
  train:
    augment: true
    augment_types:
    - HorizontalFlip
    - VerticalFlip
    - Blur
    - RandomRotate90
    - RandomBrightnessContrast
    - MultiplicativeNoise
    - Cutout
    shuffle: true
    scenes:
    - 4688749_0470419_2021-07-12_105c
    - 4691214_0470419_2021-07-14_2455
    - 4704585_1170810_2021-07-18_2427
    - 4705397_1170810_2021-07-18_2423
    - 5811537_0870113_2022-07-25_2276
    - 4691214_0470419_2021-07-14_2455
    - 4705397_1170810_2021-07-18_2423
    - 5811537_0870113_2022-07-25_2276
    - 4713120_5272315_2021-07-21_2463
    - 4770635_4572716_2021-08-06_2416
    - 3649691_4572917_2020-08-14_1039
    - 4706735_4472312_2021-07-19_1034
    - 4709512_4372018_2021-07-20_2427
    - 4751812_3973218_2021-08-01_1026
    - 4694518_0870514_2021-07-15_1064
    - '20200722_081437_1032'
    - '20200722_081438_1032'
    - 20180702_025359_0f31
    - 20180702_025400_0f31
    - 20180702_025401_0f31
    - '20180703_192559_1006'
    - '20180703_192600_1006'
    - '20180704_075856_1044'
    - '20180704_075857_1044'
    - '20180708_075756_1011'
    - '20180708_075757_1011'
    - 20180710_025407_101b
    - 20180710_025408_101b
    - 20180716_025342_0f35
    - 20180716_025343_0f35
    - 20180716_025344_0f35
    - 20180722_025446_102e
    - 20180729_193040_0f43
    - 20180729_193041_0f43
    - '20180730_025452_1012'
    - '20180801_075816_1021'
    - '20180801_075817_1021'
    - 20180825_164603_0f2a
    - 20180825_164604_0f2a
    - '20180829_025607_1034'
    - '20180829_025608_1034'
    - '20180829_025609_1034'
    - 20180921_203252_101f
    - 20180923_080226_0f34
    - 20190606_030140_0f3f
    - 20190606_030141_0f3f
    - 20190607_001259_0f44
    - 20190607_001300_0f44
    - '20190607_204203_1044'
    - '20190607_204204_1044'
    - '20190607_204205_1044'
    - 20190610_201139_0f15
    - 20190610_201140_0f15
    - '20190618_013705_23_1062'
    - '20190618_013707_30_1062'
    - 20190619_203634_0f4e
    - 20190619_203635_0f4e
    - 20190620_201330_100a
    - 20190620_201331_100a
    - 20190620_201332_100a
    - 20190621_204122_0f12
    - 20190621_204124_0f12
    - '20190622_164800_1048'
    - '20190622_164801_1048'
    - '20190622_164802_1048'
    - '20190628_030539_1027'
    - '20190628_030541_1027'
    - '20190628_030542_1027'
    - 20190628_064356_41_106a
    - 20190628_064358_48_106a
    - 20190702_050924_0f49
    - 20190702_050925_0f49
    - '20190702_064217_29_1065'
    - '20190703_201901_1006'
    - '20190703_201902_1006'
    - '20190704_172438_93_1067'
    - '20190704_172440_99_1067'
    - 20190708_201059_101b
    - 20190708_201100_101b
    - '20190713_184804_64_1067'
    - '20190713_184806_71_1067'
    - '20190714_194358_1035'
    - '20190714_194359_1035'
    - '20190714_194400_1035'
    - 20190715_000856_0f2a
    - 20190715_000857_0f2a
    - 20190716_014045_22_106a
    - '20190719_030812_1006'
    - '20190719_030813_1006'
    - '20190719_030814_1006'
    - '20190722_030702_1039'
    - '20190722_030703_1039'
    - 20190723_080935_103b
    - 20190723_080936_103b
    - 20190723_080937_103b
    - 20190723_080938_103b
    - 20190726_193719_0f17
    - 20190726_193720_0f17
    - 20190726_193721_0f17
    - '20190727_030648_1040'
    - '20190727_030649_1040'
    - '20190727_030650_1040'
    - 20190727_194259_100c
    - 20190727_194300_100c
    - '20190727_202025_1034'
    - '20190727_202026_1034'
    - '20190731_000801_1053'
    - '20190731_000802_1_1053'
    - '20190801_080518_1014'
    - '20190801_080519_1014'
    - 20190803_000717_0f49
    - 20190803_000718_0f49
    - '20190803_164435_1048'
    - '20190803_164436_1048'
    - '20190803_164437_1048'
    - '20190803_164438_1048'
    - '20190803_164439_1048'
    - '20190808_201538_1018'
    - '20190808_201539_1018'
    - 20190813_082915_68_105c
    - 20190813_082918_17_105c
    - 20190814_080303_100c
    - 20190814_080304_100c
    - '20190815_030625_1001'
    - '20190815_030626_1001'
    - 20190815_080515_0f4e
    - 20190815_080516_0f4e
    - '20190819_080913_1012'
    - '20190819_080914_1012'
    - '20190819_080915_1012'
    - '20190819_080917_1012'
    - 20190820_164142_0f21
    - 20190820_164143_0f21
    - 20190820_164144_0f21
    - 20190822_081235_101f
    - 20190822_081236_101f
    - 20190822_081237_101f
    - '20190829_030342_1006'
    - '20190829_030343_1006'
    - 20190831_030654_0f35
    - 20190831_030655_0f35
    - 20190902_203942_103b
    - 20190902_203943_103b
    - 20190903_075606_0e0f
    - 20190903_075607_0e0f
    - 20190904_193046_0e2f
    - 20190904_193047_0e2f
    - 20190904_193048_0e2f
    - '20190905_081012_1004'
    - '20190905_081013_1004'
    - '20190905_081014_1004'
    - '20190905_195022_1032'
    - '20190905_195023_1032'
    - 20190907_081306_100c
    - 20190907_081307_100c
    - 20190907_081308_100c
    - 20190907_081309_100c
    - '20190907_195118_1034'
    - '20190907_195119_1034'
    - 20190907_203545_84_105c
    - '20190913_030436_1005'
    - '20190913_030437_1005'
    - '20190916_030410_1010'
    - '20190916_030411_1010'
    - '20190921_080320_1011'
    - '20190921_080321_1011'
    - '20190928_201420_1035'
    - '20190928_201421_1035'
    - '20190928_201422_1035'
    - 5773266_0971513_2022-07-08_2439
    - 5761052_1070519_2022-07-04_241f
    - 5796176_4372315_2022-07-19_242d
    - 4737567_5268618_2021-07-28_2465
    - 5811537_0870514_2022-07-25_2276
    - 5773288_1071219_2022-07-09_248b
    - 5853699_4973918_2022-08-13_245c
    - 5851529_0870615_2022-08-12_2403
    - 5761052_1171112_2022-07-04_241f
    - 5828620_4271614_2022-08-02_2495
    - 4740689_5173016_2021-07-29_242a
    - 5803306_3971013_2022-07-22_248c
  val:
    augment: false
    shuffle: false
    scenes:
    - '20200814_044131_1026'
    - '20200814_044132_1026'
    - '20200814_044133_1026'
    - 5761052_1171112_2022-07-04_241f
  test:
    augment: false
    shuffle: false
    scenes: null
      
# Training Parameters
batch_size: 18
learning_rate: 0.001
# Learning rate scheduler. Available:
# ExponentialLR, StepLR (https://pytorch.org/docs/stable/optim.html)
# if no lr_step_size given then lr_step_size=10, gamma=0.1 for StepLR and gamma=0.9 for ExponentialLR
learning_rate_scheduler: ExponentialLR
lr_step_size: 10
lr_gamma: 0.9

# Training Schedule
schedule:
  - phase: Training
    epochs: 50
    steps:
      - train_on: train
      - validate_on: val
```

**Comment by initze** [comment]
Seems to be better now.

* check if BatchNorm is implemented
* check for functionality of Augmentation

**Comment by initze** [comment]
deprecated
